build/build_test.go

```diff
@@ -81,3 +84,51 @@ func TestWatch(t *testing.T) {
 		t.Error("expected main.js to exist")
 	}
 }
+
+func TestWatchLocalization(t *testing.T) {
+	var wg sync.WaitGroup
+	ctx, cancel := context.WithCancel(context.Background())
+
+	extension := config.Extensions[0]
+	locales_filepath := filepath.Join(".", extension.Development.RootDir, "locales")
+
+	os.RemoveAll(locales_filepath)
+	err := os.Mkdir(locales_filepath, 0775)
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
What about registering a [cleanup](https://pkg.go.dev/testing#T.Cleanup) function early so that it will be guaranteed to run?

```suggestion
	err := os.Mkdir(locales_filepath, 0775)
	if err != nil {
		t.Fatal(err)
	}
	t.Cleanup(func() {
	        os.RemoveAll(locales_filepath)
	})
```